types/proto/narwhal.proto

```diff
@@ -101,15 +63,20 @@ message GetCollectionsRequest {
     repeated CertificateDigest collection_ids = 1;
 }
 
+message GetCollectionsResponse {
+    // TODO: Revisit this for spec compliance.  
+    // List of retrieval results of collections.
+    repeated CollectionRetrievalResult result = 1;
+}
+
 message RemoveCollectionsRequest {
     // List of collections to be removed.
     repeated CertificateDigest collection_ids = 1;
 }
 
-message GetCollectionsResponse {
-    // TODO: Revisit this for spec compliance.  
-    // List of retrieval results of collections.
-    repeated CollectionRetrievalResult result = 1;
+message NewNetworkInfoRequest {
+    int32 epoch_number = 1;
```

Comment:
We might want `uint32` to stay closer to the api description?

Reply:
Good catch! Thanks